### PR TITLE
[SALEOR-6645] Allow icon and clickable props on Pill

### PR DESCRIPTION
**Pull request by magul**

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-6645 Add app URL](https://app.clickup.com/t/2549495/SALEOR-6645)

File: src/Pill/Pill.tsx [hunk 1]
```diff
@@ -12,13 +12,11 @@ export interface PillProps
     ChipProps,
     | "onDelete"
     | "avatar"
-    | "icon"
     | "disabled"
     | "deleteIcon"
     | "color"
     | "disabled"
     | "classes"
-    | "clickable"
```
Comment (Droniu) [inline src/Pill/Pill.tsx:21]: Clickable is not needed here. Clickable css class is applied by passing onClick parameter

**Review by Droniu — APPROVED** [review]

File: src/Pill/Pill.tsx (hunk 1 above)
Comment (dominik-zeglen) [reply src/Pill/Pill.tsx]: How about adding explicit class with `cursor: inherit;`?

**Comment by magul** [comment]
Based on conversation with @dominik-zeglen and @pwgryglak we will pursue different design, so I'm closing this pull request as unnecessary.

**Pull request closed without merging**